### PR TITLE
Style fix for org-unternehmen block

### LINKED ISSUE #82: geschaefsbereiche page uses Hero Images throughout
On this page: https://main--eder-group--hlxsites.hlx.page/unternehmen/geschaeftsbereiche

Images throughout the page are using the 'Hero' block.  

- [ ] replace Hero block from content with its own new block
- [ ] add correct styling for these images on the page

### PR DESCRIPTION
**Pull request by asthabh23**
Fix #82 

Test URLs:
- Before: https://main--eder-group--hlxsites.hlx.page/unternehmen/geschaeftsbereiche
- After: https://issue-82--eder-group--hlxsites.hlx.page/unternehmen/geschaeftsbereiche

### CODE REVIEW

**Comment by alexcarol** [comment]
Thanks for the fix! The image looks a bit smaller than in the original site (https://www.eder-gmbh.de/unternehmen/geschaeftsbereiche/) by like 20%, is there an easy fix for that?

**Comment by alexcarol** [comment]
The text that says "LANDTECHNIK" also looks a bit smaller.

**Comment by mhaack** [comment]
We need to give the headlines on the image more space. On bigger screen the original is always one line, ours has line break.

File: blocks/org-unternehmen/org-unternehmen.css [hunk 1]
```diff
@@ -1,56 +1,94 @@
...
+    padding: 0;
+    position: relative;
+}
+
+.section.org-unternehmen-container .org-unternehmen > div p {
+    margin: 0;
+    width: 100%;
+}
+
+.org-unternehmen div[class^='image'] p img {
```
Comment (mhaack) [inline blocks/org-unternehmen/org-unternehmen.css:16]: To keep aligned with definitions above and below we should prefix with `.section.org-unternehmen-container` here as well.

File: blocks/org-unternehmen/org-unternehmen.css [hunk 2]
```diff
@@ -1,56 +1,94 @@
...
-    max-height: 48px;
-    margin-left: auto;
-    margin-right: auto;
-    width: auto;
+    object-position: 90%;
+    padding: 0;
+    margin: 0;
+}
+
+.org-unternehmen div h3 {
```
Comment (mhaack) [inline blocks/org-unternehmen/org-unternehmen.css:28]: To keep aligned with definitions above and below we should prefix with `.section.org-unternehmen-container` here as well.

**Review by mhaack — APPROVED** [review]

**Pull request merged**